features/attached_commands.feature

```diff
@@ -4,6 +4,9 @@ Feature: Command behaviour when attached to an Ubuntu Pro subscription
     Scenario Outline: Attached refresh in a ubuntu machine
         Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
         When I attach `contract_token` with sudo
+        And I verify that `Bearer ` field is redacted in the logs
```

Comment:
gold

Reply:
:facepalm: we should've added this ages ago